metricbeat/module/coredns/_meta/kibana/7/dashboard/Metricbeat-coredns-overview.json

```diff
@@ -0,0 +1,1395 @@
+                  "disabled": false,
+                  "index": "metricbeat-*",
+                  "key": "coredns.stats.dns.request.count",
+                  "negate": false,
+                  "type": "exists",
+                  "value": "exists"
+                }
+              }
+            ],
+            "index": "metricbeat-*",
+            "query": {
+              "language": "lucene",
+              "query": ""
+            }
+          }
+        },
+        "title": "Requests by Protocol [Metricbeat CoreDNS] ECS",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [
+            {
+              "enabled": true,
+              "id": "1",
+              "params": {},
+              "schema": "metric",
+              "type": "count"
+            },
+            {
+              "enabled": true,
+              "id": "2",
+              "params": {
+                "field": "coredns.stats.proto",
+                "missingBucket": false,
+                "missingBucketLabel": "Missing",
+                "order": "desc",
+                "orderBy": "1",
+                "otherBucket": false,
+                "otherBucketLabel": "Other",
+                "size": 5
+              },
+              "schema": "segment",
+              "type": "terms"
+            }
+          ],
+          "params": {
+            "addLegend": true,
+            "addTooltip": true,
+            "isDonut": true,
+            "labels": {
+              "last_level": true,
+              "show": false,
+              "truncate": 100,
+              "values": true
+            },
+            "legendPosition": "right",
+            "type": "pie"
+          },
+          "title": "Requests by Protocol [Metricbeat CoreDNS] ECS",
+          "type": "pie"
+        }
+      },
+      "id": "ad4f1200-53e4-11e9-b466-9be470bbd327-ecs",
+      "type": "visualization",
+      "updated_at": "2019-04-25T22:47:01.782Z",
+      "version": "WzI0LDFd"
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [
+              {
+                "$state": {
+                  "store": "appState"
+                },
+                "exists": {
+                  "field": "coredns.stats.dns.request.type.count"
+                },
+                "meta": {
+                  "alias": null,
+                  "disabled": true,
+                  "index": "metricbeat-*",
+                  "key": "coredns.stats.dns.request.type.count",
+                  "negate": false,
+                  "type": "exists",
+                  "value": "exists"
+                }
+              }
+            ],
+            "index": "metricbeat-*",
+            "query": {
+              "language": "lucene",
+              "query": ""
+            }
+          }
+        },
+        "title": "Requests by Type [Metricbeat CoreDNS] ECS",
```

Comment:
this metric is also provided by server, type and zone. Instead of getting the average of those different counts, we can do some _per zone splitting_ (I think that will also remove the `success` type which comes informed from a set of metrics)